src/components/audio-player-view/audio-player-view.tsx

```diff
@@ -242,7 +244,7 @@ const AudioPlayerView = Event.withEventManager(
               {hasError ? <ErrorSlate /> : _renderView()}
               {showPluginsMenuOverlay && (
                 <PluginsMenuOverlay
-                  poster={player.sources.poster}
+                  plugins={availablePlugins}
```

Comment:
what about poster ?

Reply:
is is not nedded

Reply:
why ?

Reply:
i don’t understand your questions , it is not needed in the component, please be more specific